.github/workflows/ci-cd.yml

```diff
@@ -506,37 +506,36 @@ jobs:
         - macos-latest
         - windows-latest
         python-version:
-        - 3.11
+        - "3.11"
         - pypy-3.9
-        - 3.6
-        - 3.8
-        - 3.7
-        - 3.9
-        - >-
-          3.10
```

Comment:
I prefer the original style, which I why I'm using it everywhere. Please, don't change it.